app/templates/components/homepage-image-grid.hbs

```diff
@@ -1,28 +1,28 @@
 <div class="homepage-image-grid" ...attributes>
   <div class="homepage-image-grid__scroll-wrapper">
     <figure class="homepage-image-grid__img-short">
-      <img class="homepage-image-grid__img" src="/images/community/tinified/EmberConf19-13.jpg" alt="Ember core team members gather around the Ember logo sign at EmberConf">
+      <img class="homepage-image-grid__img" src="/images/community/tinified/EmberConf19-13.jpg" alt="Ember core team members gather around the Ember logo sign at EmberConf" role="presentation" loading="lazy">
```

Comment:
Since the Alt text is meaningful here, the role of presentation is inaccurate.

Reply:
```suggestion
      <img class="homepage-image-grid__img" src="/images/community/tinified/EmberConf19-13.jpg" alt="Ember core team members gather around the Ember logo sign at EmberConf" loading="lazy">
```